lib/python/qmk/c_parse.py

```diff
@@ -8,7 +8,7 @@
 from qmk.comment_remover import comment_remover
 
 default_key_entry = {'x': -1, 'y': 0, 'w': 1}
-single_comment_regex = re.compile(r' */[/*].*$')
+single_comment_regex = re.compile(r' */[/*] .*$')
```

Comment:
```suggestion
single_comment_regex = re.compile(r'\s+/[/*].*$')
```
What do you think about this?
Should leave URLs alone and will not require a space *after* the `//`.

And just to go on records with this, I really don't like having URLs in USB descriptors.

Reply:
That does have a lower breakage factor and seems easier to get people to accept.